Wikipedia/Code/WMFExploreSection.h

```diff
@@ -87,6 +87,12 @@ typedef NS_ENUM (NSUInteger, WMFExploreSectionType){
 @property (nonatomic, strong, readonly) CLLocation* location;
 
 /**
+ *  The placemark associated with the section, if any.
+ *
+ */
+@property (nonatomic, strong, readonly) CLPlacemark* placemark;
```

Comment:
at first, i was wondering "wait, how are we writing this to disk" then I realized `CLPlacemark` already conforms to `NSSecureCoding` :+1: